repose-aggregator/components/filters/header-user-filter/pom.xml

```diff
@@ -48,6 +48,7 @@
         <dependency>
             <groupId>org.scalatest</groupId>
             <artifactId>scalatest_${scala.MajDotMin}</artifactId>
+            <scope>test</scope>
```

Comment:
In the end, it won't matter. We're dropping the Maven build very soon. Whatever floats the boats.